data/tlg0062/tlg070/tlg0062.tlg070.perseus-eng4.xml

```diff
@@ -0,0 +1,489 @@
+
+
+<pb n="v.4.p.189"/>
+
+
+
+
+</p>
+
+<p><label>Purist</label> I see now; exchange is the use of a loose instead of a precise expression, while interchange is the use of both expressions, each in the other’s place.</p>
+
+<p><label>Lycinus</label> These subtleties are not unpleasing. Similarly, when we are concerned with a person, it is in our own interest; but when we are concerned for him, it is in his. It is true the phrases are sometimes confused, but there are those who observe the distinction; and it is as well to be on the safe side.</p>
+
+<p><label>Purist</label> Quite true.
+
+</p></div>
+
+
+<div type="textpart" subtype="section" xml:base="urn:cts:greekLit:tlg0062.tlg070.perseus-eng4:" n="11">
+<p><label>Lycinus</label> Now, can you tell me the difference between ‘setting’ and ‘sitting,’ or between ‘be seated ’ and ‘sit’?</p>
+
+<p><label>Purist</label> No; but I have heard you say that ‘sit yourself’ is a barbarism,;</p>
+
+<p><label>Lycinus</label> Yes, quite so; but now I tell you that ‘be seated ’ is not the same as ‘sit.’</p>
+
+<p><label>Purist</label> Why, what may the difference be?</p>
+
+<p><label>Lycinus</label> When a man is on his legs, you can only tell him to be seated; but if he is seated already, you can tell him to sit still.
+
+
+
+
+
+<pb n="v.4.p.190"/>
+
+
+
+
+Sit where thou art; we find us seats: elsewhere. It means ‘remain sitting,’ you see. Here again we have to say that it is a mistake to reverse the expressions. And as to
+‘set’ and ‘sit,’ surely it is the whole difference between transitive and intransitive?
+
+</p></div>
+
+
+<div type="textpart" subtype="section" xml:base="urn:cts:greekLit:tlg0062.tlg070.perseus-eng4:" n="12">
+<p><label>Purist</label> That is clear enough; go on; this is the way to teach.</p>
+
+<p><label>Lycinus</label> Or the only way you can learn? Well, do you know what a historian is?
+
+<note xml:lang="eng">The explanation of this point appears to have dropped out of
+the MSS.—Translators.</note></p>
+
+
+<p><label>Purist</label> Oh, yes, I quite see, after your lucid explanation.</p>
+
+<p><label>Lycinus</label> Now I daresay you think servility and servitude are the same; but I am aware of a considerable difference between them.</p>
+
+<p><label>Purist</label> Namely—?</p>
+
+<p><label>Lycinus</label> The first depends on yourself, the other on some one else.</p>
+
+<p><label>Purist</label> Quite right.</p>
+
+<p><label>Lycinus</label> Oh, you will pick up all sorts of information, if you give up thinking you know more than you do.</p>
+
+<p><label>Purist</label> I give it up from this moment.</p>
+
+<p><label>Lycinus</label> Then we will break off for the present, and take the rest another time H. and F.
```

Comment:
I noticed that most of the translations have H. or F. or H. and F. at the end, I imagine to indicate which or both of the brothers that translated the work,but I was wondering should that be included?